python/tvm/relay/prelude.py

```diff
@@ -35,12 +35,12 @@ def define_list_map(self):
         nil_case = Clause(PatternConstructor(self.nil), self.nil())
         cons_case = Clause(PatternConstructor(self.cons, [PatternVar(y), PatternVar(z)]),
                            self.cons(f(y), self.map(f, z)))
-        self.mod[self.map] = Function([f, x], Match(x, [nil_case, cons_case]), None, [a, b])
+        self.mod[self.map] = Function([f, x], Match(x, [nil_case, cons_case]), self.l(b), [a, b])
 
     def define_list_foldl(self):
         """Defines a left-way fold over a list.
 
-        foldl(f, z, l) : fun<a, b>(fun(b, a) -> b, b, list<a>) -> b
+        foldl(f, z, l) : fn<a, b>(fn(a, b) -> a, a, list[a]) -> a
```

Comment:
`fn<a, b>(fn(a, b) -> a, a, list[b]) -> a`?

Reply:
You're right, good catch. I'll correct it.